app/views/report/_widget_show.html.haml

```diff
@@ -36,7 +36,7 @@
           %p.form-control-static
             - unless (glyphicon = @widget.decorate.try(:secondary_icon)).nil?
               %i{:class => "#{glyphicon}"}
-            = h(@widget.status)
+            = h(_(@widget.status))
```

Comment:
Where is the content of `@widget.status` being collected for catalog inclusion?

Reply:
here ManageIQ/manageiq#20843.